lib/qesapdeployment.pm

```diff
@@ -1661,12 +1661,19 @@ sub qesap_aws_filter_query {
 
     Return the Transient Gateway ID of the IBS Mirror
 
+=over
+
+=item B<MIRROR_TAG> - Value of Project tag applied to the IBS Mirror
```

Comment:
Is it intentional for the argument name to be in uppercase?
But... I see it is like that everywhere.

Reply:
Yes it is intentional. Not sure if it is a good decision but at least intentional :-)

It is a "typographic" choice: the aim it to have argument names to stand out a little bit more.